src/index.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { createStore } from 'redux';
+import { createStore, compose, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
 import registerServiceWorker from './registerServiceWorker';
 
 import './index.css';
 import App from './App';
 
-
 import rootReducer from './reducers';
+// import updateServer from './middleware/upadteServer';
+
+/* eslint-disable no-underscore-dangle */
```

Comment:
How can we make this rule pass, without disabling eslint?

Disabling eslint is like "godmode" in games ;) It suggests that a certain part of the game is really impossible to finish otherwise, or that the player picks the easy way. Please do not pick easy ways, as you might learn important things in the process of trying to resolve things "a good way".

Reply:
I found this disabled in documentation https://github.com/zalmoxisus/redux-devtools-extension#11-basic-store. I find that I can add this  "no-underscore-dangle": ["error", { "allow": ["__REDUX_DEVTOOLS_EXTENSION_COMPOSE__"] }] to pass this rule.

Reply:
Awesome. That's what I was looking for. :)